apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -205,6 +215,29 @@ export default connect(
       }
     }
 
+    displayErrorMsgs() {
+      if (this.props.logOutput.size > 0) {
```

Comment:
What happens if there is nothing this.props.logOutput yet? (i.e. this.props.logOutput.size === 0)

Reply:
When it's `this.props.logOutput.size === 0` then nothing shows up in the `<div>` tag for debug-output:
<img width="862" alt="Screen Shot 2019-04-24 at 1 40 14 PM" src="https://user-images.githubusercontent.com/17921445/56696299-47634780-66a0-11e9-82b8-83042e956eae.png">

Then whenever the size is greater than 0 then the `<div class='debug-output'>` would expand and show the inspector tag:
<img width="878" alt="Screen Shot 2019-04-24 at 1 41 12 PM" src="https://user-images.githubusercontent.com/17921445/56696456-b8a2fa80-66a0-11e9-84a8-b327cc6592e8.png">

It seemed like whether I add a return statement of `this.props.logOutput` in the else statement or not didn't seem to impact result of the div tag for debug-output so I left it with only the if statement. Not entirely sure what other things I should consider when the size is 0. Could you point me to where I should look? Thanks for bringing up such thought provoking questions!